HW_3/Task_16.py

```diff
@@ -0,0 +1,16 @@
+import random
+N = int(input('Enter the number of elements in the array: '))
+lst = [random.randint(0, 10) for i in range(N)]
+print(lst)
+x = int(input('Enter the element you are looking for: '))
+print(x)
+lst_1 = []
+ind = 0
+for i in lst:
+    if x == lst[ind]:
+        lst_1.append(x)
+        ind += 1
+    else: ind += 1
```

Comment:
Не совсем понимаю зачем вам список `lst_1`
```suggestion
ind = 0
for i in lst:
    if x == i:
        ind += 1
```
Вот все решение. В `ind` теперь будет содержаться количество повторений

Reply:
Даниил, спасибо!